libs/math/include/math/activation_functions/sigmoid.hpp

```diff
@@ -36,23 +36,19 @@ void Sigmoid(ArrayType const &t, ArrayType &ret)
 
   auto array_it = t.cbegin();
   auto rit      = ret.begin();
-  Type zero{0};
-  Type one{1};
-  Type min_one{-1};
```

Comment:
as far as I understood, this min_one was not used at all right?

Reply:
no, it was used but i've refactored it out